lib/osf-components/addon/components/resources-list/component.ts

```diff
@@ -0,0 +1,46 @@
+import { A } from '@ember/array';
+import { assert } from '@ember/debug';
+import { waitFor } from '@ember/test-waiters';
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { task } from 'ember-concurrency';
+import { taskFor } from 'ember-concurrency-ts';
+
+import RegistrationModel from 'ember-osf-web/models/registration';
+import ResourceModel from 'ember-osf-web/models/resource';
+
+interface ResourcesListArgs {
+    registration: RegistrationModel;
+}
+
+export default class ResourcesListComponent extends Component<ResourcesListArgs> {
```

Comment:
Can we add a `reload` action to this component for reloading the list? It would be helpful for the add modal to have a `reload` action to invoke.